examples/tags/v1beta1/tagvalue.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tags.gcp.upbound.io/v1beta1
+kind: TagValue
+metadata:
+  annotations:
+    meta.upbound.io/example-id: tags/v1beta1/tagvalue
```

Comment:
```suggestion
    meta.upbound.io/example-id: tags/v1beta1/tagvalue
    upjet.upbound.io/manual-intervention: "This resource is skipped because the dependent resource is skipped"
```